docs/resources/rdb_read_replica.md

```diff
@@ -49,7 +49,32 @@ resource "scaleway_rdb_read_replica" "replica" {
   instance_id = scaleway_rdb_instance.instance.id
   private_network {
     private_network_id = scaleway_vpc_private_network.pn.id
-    service_ip         = "192.168.1.254/24" // omit this attribute if private IP is determined by the IP Address Management (IPAM)
+    service_ip         = "192.168.1.254/24"
+    # enable_ipam = false
```

Comment:
Maybe add a comment here.
If you use service_ip, the enable_ipam MUST be set to false. 
Or if by default it will be false, let's omit it all together